ADR/0027-availability-probe-framework.md

```diff
@@ -0,0 +1,102 @@
+# 27. Availability Probe Framework
+
+* Date 2023-07-06
+
+## Status
+
+Accepted
+
+## Context
+
+As a product team, we want a composite system availability metric for both Production
+and Staging environments, so that we can make roadmap decisions with easy-to-understand
+feedback about the reliability of the system in mind.
```

Comment:
This might not make as much sense within the context of AppStudio itself. Maybe something like 

```suggestion
As an AppStudio developer building functionality for the platform, I want to be able to easily
visualize and comprehend the stability and availability of deployed systems in order to inform
and influence future work towards improving the overall system reliability.
```